### DISCUSSION TITLE
en/chapter_computational_complexity/iteration_and_recursion/

### DISCUSSION THREAD

**Discussion by giscus** [krahets/hello-algo#1480] [Announcements]
# en/chapter_computational_complexity/iteration_and_recursion/

Data Structures and Algorithms Crash Course with Animated Illustrations and Off-the-Shelf Code

https://www.hello-algo.com/en/chapter_computational_complexity/iteration_and_recursion/

**Comment by Bonsrah** [comment]
This took me too long to understand, eventually did.
appreciate the lesson

**Comment by flasheddy** [comment]
By coding along with the lesson, I understood iteration and recursion better. If I finish the stack lesson, I'll come back here later to get a deeper understanding.
